src/app/interfaces.ts

```diff
@@ -2,4 +2,5 @@ export interface Movie {
   type: 'youtube' | 'vimeo';
   id: string;
   link: string;
+  loading: boolean;
```

Comment:
zrób to jako opcjonalne property, żeby funkcja parsująca nie musiala tego dodawać w komponencie bo to slabe miejsce na to. loading?: boolean to jest zapis opcjonalnej property, wtedy możejej nie być albo może być o zadanym typoe